code/_global_vars/lists/objects.dm

```diff
@@ -15,8 +15,19 @@ var/global/host = null //only here until check @ code\modules\ghosttrap\trap.dm:
 var/global/datum/sun/sun = new
 var/global/datum/universal_state/universe = new
 
-var/global/list/vowels =             list("a","e","i","o","u")
-var/global/list/alphabet_no_vowels = list("b","c","d","f","g","h","j","k","l","m","n","p","q","r","s","t","v","w","x","z")
-var/global/list/full_alphabet =      list("a","b","c","d","e","f","g","h","i","j","k","l","m","n","o","p","q","r","s","t","u","v","w","x","y","z")
+/// Vowels.
+var/global/list/vowels =                     list("a","e","i","o","u")
+/// Alphabet a-z.
+var/global/list/alphabet =                   list("a","b","c","d","e","f","g","h","i","j","k","l","m","n","o","p","q","r","s","t","u","v","w","x","y","z")
+/// Alphabet A-Z.
+var/global/list/alphabet_capital =           list("A","B","C","D","E","F","G","H","I","J","K","L","M","N","O","P","Q","R","S","T","U","V","W","X","Y","Z")
+/// Alphabet a-z without vowels.
+var/global/list/alphabet_vowelless =         list("b","c","d","f","g","h","j","k","l","m","n","p","q","r","s","t","v","w","x","z")
+/// Alphabet A-Z without vowels.
+var/global/list/alphabet_vowelless_capital = list("B","C","D","F","G","H","J","K","L","M","N","P","Q","R","S","T","V","W","X","Z")
```

Comment:
What are these for? This list is redundant in view of `uppertext(alphabet_vowelless)` and numbers_alt isn't used at all that I can see.